ibm_i/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "display_name": "IBM i",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "ibm_i",
+  "metric_prefix": "ibm_i.",
+  "metric_to_check": "ibm_i.system.cpu_usage",
+  "creates_events": false,
+  "short_description": "Remotely monitor IBM i systems including jobs, job queues, ASPs and more.",
+  "guid": "da389374-7541-47e5-bcd1-87cf3b88a469",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "IBM i",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": false,
```

Comment:
update to `true` when the integration is publicly available (affects documentation)

Reply:
Got it, we will do in a follow up PR